From e857f4e21a4bebe5187cc05f047f70450bd8193a Mon Sep 17 00:00:00 2001
From: Marcel Metz <mmetz@adrian-broher.net>
Date: Tue, 8 Mar 2016 10:56:04 +0100
Subject: [PATCH 10/10] CMake: Set install MacOSX Framework location to
 "Frameworks"

Users should be able to install the MacOSX Framework by pointing
CMAKE_INSTALL_PREFIX to the corresponding directory. For example to
install into `/Library/Frameworks` configure with

cmake -DCMAKE_INSTALL_PREFIX=/Library <build dir>
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 407b901..fbfd864 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -110,7 +110,7 @@ install(TARGETS ogg
     RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
     LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
     ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
-    FRAMEWORK DESTINATION ${CMAKE_INSTALL_LIBDIR}
+    FRAMEWORK DESTINATION "Frameworks"
     PUBLIC_HEADER DESTINATION ${CMAKE_INSTALL_FULL_INCLUDEDIR}/ogg
 )
 if(NOT WIN32 OR CYGWIN)
-- 
2.5.0

